tools/webpack.config.babel.js

```diff
@@ -1,15 +1,27 @@
 const path = require('path')
```

Comment:
why did you call the file webpack.config.babel.js and not just webpack.config.js ? Afaics it's just a normal webpack config. (Same for lib)

Reply:
See other answer